components/automate-ui/src/app/pages/service-groups/service-groups.component.ts

```diff
@@ -67,8 +67,8 @@ export class ServiceGroupsComponent implements OnInit, OnDestroy {
   private selectedSortField$: Observable<string>;
   private healthSummary$: Observable<HealthSummary>;
   private currentPage$: Observable<number>;
-  private currentFieldDirection: SortDirection;
-  private currentSortField: string;
+  public currentFieldDirection: SortDirection;
+  public currentSortField: string;
```

Comment:
Interesting to note that these were used in some unit tests--even though they were private!--with no complaint in our current environment. But the upgrade will get more particular, requiring this fix.